src/Controller/HomeController.php

```diff
@@ -12,8 +13,16 @@ class HomeController extends AbstractController
      */
     public function index()
     {
+        $finder = new Finder();
```

Comment:
Originalu.

Bet rekomenduojamas būdas yra naudoti `KernelInterface`.
Arba bent jau `PHP` standartinėje bibliotekoje esantį `file_get_contents(__DIR__ . '../../public/students.json');`

Reply:
Šiaip gera praktika – jei įmanoma darbą atlikti be papildomų bibliotekų – geriau jų ir nenaudoti.
Nes kiekviena papildoma biblioteka reiškia, kad reikės kada nors ją atnaujinti, sužiūrėti licenzijas, saugumo skyles, apsaugoti nuo jos ištrynimo ir pan.

Daugumai paprastų dalykų jau yra standartinės karkaso (pvz. `Symfony`) arba kalbos (pvz. `PHP`) funkcijos.

Reply:
Ačiū už pastebėjimus :)